.github/workflows/system_tests.yml

```diff
@@ -1,138 +1,72 @@
-        kube_config: ${{ secrets[format('DEV{0}KC', env.RUNNER_NUMBER)] }}
-        teardown_condition: "TESTS_PASSED"
-        miner_image:  ${{ github.event.inputs.miner_image }}
-        sharder_image: ${{ github.event.inputs.sharder_image }}
-        blobber_image: ${{ github.event.inputs.blobber_image }}
-        validator_image: ${{ github.event.inputs.validator_image }}
-        zbox_image: ${{ github.event.inputs.zbox_image }}
-        zblock_image: ${{ github.event.inputs.zblock_image }}
-        zdns_image: staging
-        explorer_image: latest
-        zproxy_image: staging
-        zsearch_image: staging
-        blobber_stake_image: latest
-
-    - name: "Run System tests"
-      if: ${{ github.event.inputs.skip_tests == 'FALSE' }}
-      uses: 0chain/actions/run-system-tests@master
-      with:
-        custom_go_sdk_version: ${{ env.CURRENT_BRANCH_HEAD }}
-        system_tests_branch: ${{ github.event.inputs.system_tests_branch }}
-        network: ${{ env.NETWORK_URL }}
-        zbox_cli_branch: ${{ github.event.inputs.zbox_cli_branch }}
-        zwallet_cli_branch: ${{ github.event.inputs.zwallet_cli_branch }}
-        smart_contract_owner_wallet_json: ${{ secrets.SMART_CONTRACT_OWNER_WALLET_JSON }}
-        svc_account_secret: ${{ github.token }}
-        deploy_report_page: false
-        archive_results: true
-        run_flaky_tests: false
-        retry_failures: true
-        
-    - name: "Set PR Status Success"
-      if: ${{ success() }}
-      uses: 0chain/actions/notify-pull-request@feature/notify-PR-convenience
-      continue-on-error: true
-      with:
-        state: ${{ job.status }}
-        message: "manual tests failed"
-        system_tests_branch: ${{ github.event.inputs.system_tests_branch }}
-        zbox_cli_branch: ${{ github.event.inputs.zbox_cli_branch }}
-        zwallet_cli_branch: ${{ github.event.inputs.zwallet_cli_branch }}
-        gosdk_branch: ${{ env.CURRENT_BRANCH_HEAD }}
-        zerochain_branch: ${{ github.event.inputs.miner_image }}
-        blobber_branch: ${{ github.event.inputs.blobber_image }}
-        github_token: ${{ secrets.SVC_ACCOUNT_SECRET }}
-
-    - name: "Set PR Status Failed"
-      if: ${{ failure() }}
-      uses: 0chain/actions/notify-pull-request@feature/notify-PR-convenience
-      continue-on-error: true
-      with:
-        state: ${{ job.status }}
-        message: "manual tests success"
-        system_tests_branch: ${{ github.event.inputs.system_tests_branch }}
-        zbox_cli_branch: ${{ github.event.inputs.zbox_cli_branch }}
-        zwallet_cli_branch: ${{ github.event.inputs.zwallet_cli_branch }}
-        gosdk_branch: ${{ env.CURRENT_BRANCH_HEAD }}
-        zerochain_branch: ${{ github.event.inputs.miner_image }}
-        blobber_branch: ${{ github.event.inputs.blobber_image }}
-        github_token: ${{ secrets.SVC_ACCOUNT_SECRET }}
+  system-tests:
+    uses: 0chain/actions/.github/workflows/manual_system_tests.yml@feature/notify-PR-convenience
```

Comment:
we will always use `feature/notify-PR-convenience` instead of `master`?

Reply:
nope, once https://github.com/0chain/actions/pull/8 has merged to master we can change this to master